components/fonts/system_font_service.rs

```diff
@@ -84,9 +88,12 @@ struct ResolvedGenericFontFamilies {
 /// The system font service. There is one of these for every Servo instance. This is a thread,
 /// responsible for reading the list of system fonts, handling requests to match against
 /// them, and ensuring that only one copy of system font data is loaded at a time.
+#[derive(MallocSizeOf)]
 pub struct SystemFontService {
+    #[ignore_malloc_size_of = "ipc-channel type"]
```

Comment:
We already have:

```rust
impl<T> MallocSizeOf for ipc_channel::ipc::IpcSender<T> {
    fn size_of(&self, _ops: &mut MallocSizeOfOps) -> usize {
        0
    }
}
```

so I think it makes sense to do the same for `IpcReceiver` instead of adding an exception here.